### DISCUSSION TITLE
In case of web-app, it is accessible via xxx from other apps.

### DISCUSSION THREAD

**Discussion by 7aklhz** [caprover/caprover#982] [Q&A] [answered]
Hi, 

I have 2 apps : 
 - 1 app running a php server
 - 1 app running a static website. 

The static website calls the php server to retrieve information via a POST. I would like to avoid making the php server public and thus access the server "internally"/between my 2 apps running on caprover. 

In HTTP settings, it's written "In case of web-app, it is accessible via http://srv-captain--<app name> from other apps". As such, I tried using this url in the app running the static website like this: 

```
axios
.get("http://srv-captain--<app name>", {})
.then(response => {
                                console.log(" -> Success")
                                    resolve(response.data)
                                })
.catch(e => {

                                    console.log("error " + e)
                                })
})
```

However, I'm getting the following error :  net::ERR_NAME_NOT_RESOLVED

Can someone explain to my if I'm calling the correct URL and/or if generally speaking I'm doing this correctly ?

Thank you

el_even

**Comment by githubsaturn** [comment, marked as answer]
The frontend code is running the browser, not on the server. `srv-captain--appname` is only available from within the containers.

**Reply by 7aklhz** [reply]
Ok, thank you for your reply. Cheers.
